rllib/algorithms/tests/test_algorithm.py

```diff
@@ -160,8 +160,7 @@ def new_mapping_fn(agent_id, episode, worker, **kwargs):
 
                 # Test restoring from the checkpoint (which has more policies
                 # than what's defined in the config dict).
-                test = pg.PG(config=config)
-                test.restore(checkpoint)
+                test = pg.PG.from_checkpoint(checkpoint)
```

Comment:
New way of restoring an algo w/o needing the config. The old way would still work as well, though.